packages/plugins/rubygem-foreman_statistics/rubygem-foreman_statistics.spec

```diff
@@ -83,11 +98,15 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/db
 %{gem_libdir}
 %{gem_instdir}/locale
+%exclude %{gem_instdir}/package.json
+%exclude %{gem_instdir}/webpack
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
 %{foreman_apipie_cache_foreman}
 %{foreman_apipie_cache_plugin}
```

Comment:
You also need `%{foreman_assets_plugin}` to include the  `public/assets/foreman_statistics` folder - as the compile places a json file there